scripts/screeps_tencent_batch_rl_runner.py

```
@@ -245,6 +251,29 @@ def scp_from_worker(self, name: str, remote_path: str, local_path: Path, *, time
             timeout=timeout,
         )
 
+    def clear_worker_known_host(self) -> None:
+        if not self.public_ip:
+            return
+        if self.public_ip in self.known_hosts_cleaned_public_ips:
+            return
+        cmd = ["ssh-keygen", "-R", self.public_ip, "-f", str(self.known_hosts_path)]
+        started = time.time()
+        try:
+            cp = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)